content/src/content/jcr_root/apps/core/wcm/components/carousel/v1/carousel/carousel.html

```diff
@@ -80,7 +80,7 @@
             <li data-sly-repeat.item="${carousel.items}"
                 class="cmp-carousel__indicator${itemList.first ? ' cmp-carousel__indicator--active' : ''}"
                 role="tab"
-                aria-controls="${carousel.id}-item-${itemList.index}"
+                aria-controls="${item.id}-tabpanel"
```

Comment:
Should we also add an id here? `id="${item.id}-tab"`, for consistency?

Reply:
Yes, added.